pkg/secrethub/service.go

```diff
@@ -54,25 +54,20 @@ func (s serviceService) Create(path string, description string, credentialCreato
 		return nil, errio.Error(err)
 	}
 
-	credentialRequest, err := s.client.createCredentialRequest(credentialCreator.Verifier(), credentialCreator.Metadata())
+	credentialRequest, err := s.client.createCredentialRequest(credentialCreator.Encrypter(), accountKey, credentialCreator.Verifier(), credentialCreator.Metadata())
 	if err != nil {
 		return nil, errio.Error(err)
 	}
 
-	accountKeyRequest, err := s.client.createAccountKeyRequest(credentialCreator.Encrypter(), accountKey)
-	if err != nil {
-		return nil, errio.Error(err)
-	}
-
-	serviceRepoMemberRequest, err := s.client.createRepoMemberRequest(repoPath, accountKeyRequest.PublicKey)
+	serviceRepoMemberRequest, err := s.client.createRepoMemberRequest(repoPath, credentialRequest.AccountKey.PublicKey)
 	if err != nil {
 		return nil, errio.Error(err)
 	}
 
 	in := &api.CreateServiceRequest{
 		Description: description,
 		Credential:  credentialRequest,
-		AccountKey:  accountKeyRequest,
+		AccountKey:  credentialRequest.AccountKey,
```

Comment:
Aren't we now sending the same account key request twice? I believe we can remove the separate `CreateServiceRequest.AccountKey` altogether. Of course, for backwards compatibility, the server should still accept it though.